src/forms/FormNav/FormNavTypes.tsx

```diff
@@ -5,4 +5,5 @@ export interface Section {
 
 export interface FormNavProps {
   sections: Section[];
+	sectionsRefs?: HTMLDivElement[] | []

```

Comment:
The prop has some weird indentation,  I'll change this here if there's nothing else to change / improve.